docs/developers-clear-containers-install.md

```diff
@@ -124,7 +124,7 @@ $ sudo mkdir -p /etc/systemd/system/docker.service.d/
 $ cat << EOF | sudo tee /etc/systemd/system/docker.service.d/clear-containers.conf
 [Service]
 ExecStart=
-ExecStart=/usr/bin/dockerd -D --add-runtime clearcontainers=/usr/local/bin/cc-runtime --default-runtime=runc
+ExecStart=/usr/bin/dockerd -D --add-runtime cc-runtime=/usr/local/bin/cc-runtime --default-runtime=runc
```

Comment:
This is a manual step in fedora and ubuntu, since we do not want to mess up with user own configuration. 
Step 3 in the installation docs, so if we are going to change the name of the runtime for `cc-runtime` better change also in these two docs. 

https://github.com/clearcontainers/runtime/blob/master/docs/fedora-installation-guide.md
https://github.com/clearcontainers/runtime/blob/master/docs/ubuntu-installation-guide.md